test/hw16/login_and_added_to_cart_test.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library         ../../src/hw16/helper/LoginHelper.py
+Suite Setup     Open The Browser And Go To The Home Page
+Test Setup      Login SetUp
```

Comment:
Login SetUp should be a test setup for Added to Cart test only
This keyword contains steps which should be Login test steps, not a precondition